sdk/README.md

```diff
@@ -133,86 +130,21 @@ in the same directory:
 
 After compiling the `sdk/python/tests/compiler/testdata/parallel_join.py` DSL script
 in the step above, we need to deploy the generated Tekton YAML to our Kubernetes
-cluster with `kubectl`. The Tekton server will automatically start a pipeline run
-for which we can follow the logs using the `tkn` CLI. 
+cluster with `kubectl`. The Tekton server will automatically start a pipeline run.
+We can follow the logs using the `tkn` CLI.
 
-Here we have to deploy the pipeline in the kubeflow namespace because all the pipelines with metadata
-and artifacts tracking  rely on the minio object storage credentials in the kubeflow namespace.
-
-    kubectl apply -f pipeline.yaml -n kubeflow
+    kubectl apply -f pipeline.yaml
     
-    tkn pipelinerun logs --last -n kubeflow
+    tkn pipelinerun logs --last --follow
 
 Once the Tekton Pipeline is running, the logs should start streaming:
       
     Waiting for logs to be available...
     
     [gcs-download : main] With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment makes them desolate
 
-    [gcs-download : copy-artifacts] Added `storage` successfully.
-    [gcs-download : copy-artifacts] tar: removing leading '/' from member names
-    [gcs-download : copy-artifacts] tekton/results/data
-    [gcs-download : copy-artifacts] `data.tgz` -> `storage/mlpipeline/artifacts/parallel-pipeline/gcs-download/data.tgz`
-    [gcs-download : copy-artifacts] Total: 0 B, Transferred: 195 B, Speed: 1 B/s
-
     [gcs-download-2 : main] I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my presumption not provoke thy wrath
 
-    [gcs-download-2 : copy-artifacts] Added `storage` successfully.
-    [gcs-download-2 : copy-artifacts] tar: removing leading '/' from member names
-    [gcs-download-2 : copy-artifacts] tekton/results/data
-    [gcs-download-2 : copy-artifacts] `data.tgz` -> `storage/mlpipeline/artifacts/parallel-pipeline/gcs-download-2/data.tgz`
-    [gcs-download-2 : copy-artifacts] Total: 0 B, Transferred: 205 B, Speed: 1 B/s
-
-    [echo : main] Text 1: With which he yoketh your rebellious necks Razeth your cities and subverts your towns And in a moment makes them desolate
-    [echo : main]
-    [echo : main] Text 2: I find thou art no less than fame hath bruited And more than may be gatherd by thy shape Let my presumption not provoke thy wrath
-    [echo : main]
-
-
-## Building Tekton from Master
-
-In order to utilize the latest features and functions of the `kfp-tekton` compiler,
-we suggest to install Tekton from a nightly built or build it from the
-[master](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md#install-pipeline)
-branch. Features that require a special build, different from the 'Tested Version',
-will be listed below.
-
-- [Exit Handler](/sdk/FEATURES.md#exit-handler)
-
-
-## Optional Features
-
-### Compile Kubeflow Pipelines without Artifacts
```

Comment:
We are removing this because this moved to server side 
cc @Tomcli

Reply:
yes, I discussed with Christian and this is no longer needed (for SDK).

Reply:
we originally had this section for:
- generate optional PipelineRun ... no longer optional
- enable (later disable) artifacts ... server side now